src/main/java/mate/academy/springboot/swagger/dto/request/ProductRequestDto.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.springboot.swagger.dto.request;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter

```

Comment:
use @Date here

Reply:
I do not need equals(), hashcode(), toString() in Dto 
just getters and setters for setting to needed fields

Reply:
ok. As you like)